syntax = "proto3";

package myapp;

service FileService {
  rpc UploadFile(UploadFileRequest) returns (UploadFileResponse) {}
  rpc DownloadFile(DownloadFileRequest) returns (DownloadFileResponse) {}
}

message UploadFileRequest {
  string file_name = 1;
  bytes file_content = 2;
}

message UploadFileResponse {
  bool success = 1;
  string message = 2;
}

message DownloadFileRequest {
  string file_name = 1;
}

message DownloadFileResponse {
  bytes file_content = 1;
}
